index.html

```diff
@@ -1,112 +1,199 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta name="description" content="">
-    <meta name="author" content="">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
-    <title>Géoportail LiDAR du SITN</title>
+<head>
+  <meta charset="utf-8">
+  <meta name="description" content="">
+  <meta name="author" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+  <link rel="icon" href="favicon.ico" />
+  <title>Potree Viewer</title>
```

Comment:
```suggestion
  <title>Géoportail LiDAR du SITN</title>
```
`<title>` is often used in SEO, would like to keep LiDAR and SITN together there :)

Reply:
Sorry I missed this one!